.prow/applications-catalog.yaml

```diff
@@ -0,0 +1,640 @@
+presubmits:
+  - name: argocd
```

Comment:
Job names should match the usual pattern we use, so e.g. this could/should be `pre-kubermatic-appcatalog-argocd`.